pipeline/metadata/6.0.yaml

```diff
@@ -214,3 +214,147 @@
     - ibmc
     - rgw
     - stage-1
+- name: "Tier-0 Fs"
```

Comment:
IMO, we should provide a more descriptive name so that stakeholders can identify with what is/has been tested.